app/views/pages/cookies.md

```diff
@@ -1,126 +1,140 @@
+| `_vis_opt_out`                                | This VWO cookie indicates that you should not be made part of any campaign                                                                                                                                                                               | 10 years                                        |
+| `_vwo_uuid`                                   | This VWO cookie generates a unique ID for every visitor                                                                                                                                                                                                  | 10 years                                        |
+| `_vwo_ds`                                     | This VWO cookie stores persistent visitor-level data for VWO Insights                                                                                                                                                                                    | Depends on the retention period of the account  |
+| `_vwo_sn`                                     | This VWO cookie stores session-level information                                                                                                                                                                                                         | 30 minutes and reset again to 30 minutes on activity  |
+| `_vis_opt_exp_{CAMPAIGNID}_combi_choose`      | This VWO cookie stores the chosen combination. It's a temporary cookie that gets deleted when the campaign changes are applied                                                                                                                           | Persists until campaign changes are applied     |
+| `_vwo_referrer`                               | This cookie stores referral info allowing VWO to identify the original traffic source in case of the split URL campaign                                                                                                                                  | 15 seconds                                         |
+| `_vvwo`                                       | This VWO cookie is a jar of all the VWO cookies. It is currently created only for the Safari browser                                                                                                                                                     | Expires in 2096 years                           |
+
+## Tracking and performance cookies
+
+**Type:** Persistent cookies
+
+**Administered by:** Third parties
+
+**Purpose:** These cookies are used to track information about traffic to the website and how users use the website.  We measure website usage so that we can gain a better understanding of how people use our site. This enables us to improve the information that we provide. It also ensures that the service is available when you want it, and that it delivers information quickly.
```

Comment:
```suggestion
**Purpose:** These Cookies are used to track information about traffic to the Website and how users use the Website.  We measure Website usage so that we can gain a better understanding of how people use our site. This enables us to improve the information that we provide. It also ensures that the service is available when you want it, and that it delivers information quickly.
```